src/lib/Scenes/Artwork/__tests__/Artwork-tests.tsx

```diff
@@ -21,7 +22,7 @@ describe("Artwork", () => {
   })
   it("renders a snapshot", () => {
     const component = mount(
-      <Artwork isVisible artwork={ArtworkFixture} relay={{ environment: {} } as RelayRefetchProp} />
+      <Artwork isVisible artwork={ArtworkFixture as any} relay={{ environment: {} } as RelayRefetchProp} />
```

Comment:
I'm not happy about it but was hitting a wall with a typescript issue for these. 😭

Reply:
I was taking a look at this today also, the mock Artwork is incorporating too many extra fields, passed on it for now as it seems a bit hairy.